app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -316,6 +333,7 @@ class SettingsFragment : PreferenceFragmentCompat(), CoroutineScope, AccountObse
     override fun onAuthenticated(account: OAuthAccount) {
         updateAuthState(account)
         updateSignInVisibility()
+        displayAccountErrorIfNecessary()
```

Comment:
At this point, you should be in a good state - so you likely don't need this call.